packages/hoppscotch-common/src/pages/index.vue

```diff
@@ -71,13 +71,41 @@
       @hide-modal="onCloseConfirmSaveTab"
       @resolve="onResolveConfirmSaveTab"
     />
-    <HoppSmartConfirmModal
-      :show="confirmingCloseAllTabs"
-      :confirm="t('modal.close_unsaved_tab')"
-      :title="t('confirm.close_unsaved_tabs', { count: unsavedTabsCount })"
-      @hide-modal="confirmingCloseAllTabs = false"
-      @resolve="onResolveConfirmCloseAllTabs"
-    />
+    <HoppSmartModal
+      v-if="confirmingCloseForTabID !== null"
+      :title="t('modal.close_unsaved_tab')"
+      @close="confirmingCloseForTabID = null"
+    >
+      <template #body>
+        <div class="text-center">
+          {{ t("confirm.save_unsaved_tab") }}
+        </div>
+      </template>
+      <template #footer>
+        <span class="flex space-x-2">
+          <HoppButtonPrimary
+            v-focus
+            :label="t?.('action.save')"
+            outline
+            @click="onResolveConfirmSaveTab"
+          />
+          <HoppButtonSecondary
+            :label="t?.('action.dont_save')"
+            filled
+            outline
+            @click="onCloseConfirmSaveTab"
+          />
+        </span>
+        <span class="flex space-x-2">
```

Comment:
An extra cancel button is not required. Let the cross icon on the top right of the modal handle the cancellation case

Reply:
@shipko The cross button inherently means to cancel the action. The same design and implementation is followed for most modals in the app. So we will follow the same. Let's remove the cancel button as it's redundant.

Reply:
@JoelJacobStephen done!